lib/main.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:qrapp/services/qrservice.dart';
+import 'package:qrapp/ui/pages/home_page.dart';
+import 'package:qrapp/ui/pages/qrgen_page.dart';
+import 'package:qrapp/ui/pages/qrscanner_page.dart';
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+  runApp(const QRApp());
+}
+
+class QRApp extends StatelessWidget {
+  const QRApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MultiProvider(
+      providers: [
+        ChangeNotifierProvider.value(value: QRService()),
```

Comment:
**bug**: this is wrong, you should't use the `.value()` named constructor like that! If you want a reference, look at the [pub](https://pub.dev/packages/provider#exposing-a-value) documentation. Here's what I am referring to:

<img width="436" alt="pub" src="https://user-images.githubusercontent.com/79466623/141304247-6dd029e8-4cb6-4f41-8933-0847b340b496.png">